react-native-reanimated-tests.tsx

```diff
@@ -464,3 +465,11 @@ function interpolateColorTest() {
 
   return null;
 }
+
+function interpolateNodeTest() {
+  const value = useValue(0);
+  interpolateNode(value, {
+    inputRange: [0, 1],
+    outputRange: ['0%', '100%'],
```

Comment:
We only support interpolating degrees in the string form.

```suggestion
    outputRange: ['0deg', '360deg'],
```